### ISSUE TITLE
Run Sublime with dark GTK theme

### ISSUE THREAD

**Issue by Gotterbild** [snapcrafters/sublime-text#26]
TL;DR; Is there a way to use dark GTK theme for Sublime?

I'm running Ubuntu 18.04 with default GNOME and Adwaita theme. Adwaita theme uses light colors for backgrounds. Most of apps use lighter window header and menu backgrounds. Example of app with lighter layout is `gedit`.

However, some apps still have dark layout (window header and menu). Example of such app is `gnome-terminal`. It is good that terminal uses darker theme as its window content is dark.

I use Monokai sublime color theme, which is dark. Also, I use adaptive theme to make sidebar dark as well. But Sublime's window header and menu background inherits Adwaita colors and is light. It looks ugly. I would post a screenshot with pleasure, but looks like this feature is disabled in this repo as I can't do that.

I want Sublime Text to use dark colors for window header and menu. When I run
`GTK_THEME=Adwaita-dark subl`
menu background becomes dark, but window header doesn't.

I've tried to run snap as a shell script
`GTK_THEME=Adwaita-dark snap run --shell sublime-text.subl`
but nothing happens. Sublime text doesn't start.

So, the question is - is there a way to run Sublime Text with dark window header and menu even if lighter theme is used? The same way as it works `gnome-terminal`.

If there is possibility to ship Sublime's snap with this by default? I think most Sublime users stick with Sublime's default Monokai color theme, which is dark, so most people would appreciate it. 

As an alternative, there could be some flag to run `subl` with or other way to enable dark GTK theme.

However, a bash command to run sublime with dark theme will be enough for me.

Thanks a lot!

**Comment by Gotterbild** [comment]
After some digging I think that there's a chance that it is not GTK who is drawing window header. 

When I run
`GTK_DEBUG=interactive subl`
there is GTK inspector window available for debugging. 

When in inspector I click on `GtkWindow` (which is top level element) - whole window starts blinking, except window header. 

Not sure if that info helps.

**Comment by om26er** [comment]
GNOME discourages application theming due to all rendering issues that arise. And the link that you mentioned contains a comment from @ebassi, which specifically discourages against that.

OTH: for snap package, I believe this is not something that we'd be supporting.

**Comment by om26er** [comment]
> I've tried to run snap as a shell script
> `GTK_THEME=Adwaita-dark snap run --shell sublime-text.subl`
> but nothing happens. Sublime text doesn't start.

To start it, you need to remove the `--shell` flag, that flag is for debugging and opens a shell environment of a snap's context.
```
GTK_THEME=Adwaita-dark snap run sublime-text.subl
```

**Comment by merlijn-sebrechts** [comment]
Closing this issue as "wontfix". Users who want this can create a custom desktop file or run it from the terminal as @om26er showed.
